cli-typescript/src/abis/index.ts

```diff
@@ -0,0 +1,386 @@
+export const SET_TRANSFER_VALIDATOR_ABI = {
+  inputs: [
+    {
+      internalType: 'address',
+      name: 'transferValidator_',
+      type: 'address',
+    },
+  ],
+  name: 'setTransferValidator',
+  outputs: [],
+  stateMutability: 'nonpayable',
+  type: 'function',
+} as const;
+
+export const APPLY_LIST_TO_COLLECTION_ABI = {
+  inputs: [
+    {
+      internalType: 'address',
+      name: 'collection',
+      type: 'address',
+    },
+    {
+      internalType: 'uint120',
+      name: 'id',
+      type: 'uint120',
+    },
+  ],
+  name: 'applyListToCollection',
+  outputs: [],
+  stateMutability: 'nonpayable',
+  type: 'function',
+} as const;
+
+export const IS_SETUP_LOCKED_ABI = {
+  inputs: [],
+  name: 'isSetupLocked',
+  outputs: [
+    {
+      internalType: 'bool',
+      name: '',
+      type: 'bool',
+    },
+  ],
+  stateMutability: 'view',
+  type: 'function',
+} as const;
+
+export const SUPPORTS_INTERFACE_ABI = {
+  inputs: [
+    {
+      name: 'interfaceId',
+      type: 'bytes4',
+      internalType: 'bytes4',
+    },
+  ],
+  name: 'supportsInterface',
+  type: 'function',
+  stateMutability: 'view',
+  outputs: [
+    {
+      name: '',
+      type: 'bool',
+      internalType: 'bool',
+    },
+  ],
+} as const;
+
+export const NEW_CONTRACT_INITIALIZED_EVENT_ABI = {
+  name: 'NewContractInitialized',
+  type: 'event',
+  inputs: [
+    { name: 'contractAddress', type: 'address' },
+    { name: 'creator', type: 'address' },
+    { name: 'implId', type: 'uint32' },
+    { name: 'standardId', type: 'uint8' },
+    { name: 'name', type: 'string' },
+    { name: 'symbol', type: 'string' },
+  ],
+} as const;
+
+export const ERC721mAbis = {
```

Comment:
nit: Let's pick a variable naming style for all ABIs and stick to it. IE, `TYPICAL_CONST_STYLE` or `camelCase`.

Reply:
Fixed!